posrednik.js

```diff
@@ -21,11 +21,12 @@ class Posrednik extends Client {
         super({
             clientOptions: {},
             prefix: 'p!',
-            cmdPrompt: true,
-            cmdEditing: true,
+            commandPrompt: true,
+            commandEditing: true,
             // Grabs the newly set permission levels from up top
             permissionLevels
         });
+        this.methods = { Embed: require('discord.js').MessageEmbed };

```

Comment:
1st. `methods` was removed from client for a reason, the prop isnt related to the instance of the client it doesnt make sense to have it assigned to the instance.
2nd. discord.js should be required outside the constructor

Reply:
Fixed